multidim-interop/js/v0.41/package.json

```
@@ -1,30 +1,38 @@
 {
-    "name": "multidim-interop",
-    "private": true,
-    "version": "1.0.0",
-    "description": "Multidimension Interop Test",
-    "type": "module",
-    "main": "index.js",
-    "author": "Glen De Cauwsemaecker <glen@littlebearlabs.io>",
-    "license": "MIT",
-    "engines": {
-        "node": ">=16"
-    },
-    "scripts": {
-        "start": "node index.js",
-        "test": "standard"
-    },
-    "dependencies": {
-        "@chainsafe/libp2p-noise": "^10.2.0",
-        "@chainsafe/libp2p-yamux": "^3.0.4",
-        "@libp2p/mplex": "^7.1.1",
-        "@libp2p/tcp": "^6.0.8",
-        "@libp2p/websockets": "^5.0.1",
-        "@multiformats/multiaddr": "^11.1.4",
-        "libp2p": "^0.41.0",
-        "redis": "4.5.1"
-    },
-    "devDependencies": {
-        "standard": "^17.0.0"
-    }
-}
+  "name": "multidim-interop",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Multidimension Interop Test",
+  "type": "module",
+  "main": "index.js",
+  "author": "Glen De Cauwsemaecker <glen@littlebearlabs.io>",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16"
```

might want to change this to the following to ensure fetch works

```suggestion
    "node": ">=18"
```